lucas/lucas.py

```diff
@@ -0,0 +1,83 @@
+"""This module contains an iterator class and a generator function for providing the lucas number sequence."""
+
+
+import time
+
+
+class MyLucas:
+    """This class creates an iterator which returns the lucas sequence of numbers."""
+
+    def __init__(self, max_no):
+        """Initialise MyLucas class with a maximum sequence value."""
+        self.max_no = max_no
+
+    def __iter__(self):
+        """Inititialise first two lucas numbers and return iterator object.
+
+        Returns
+        ------
+        obj
```

Comment:
So `object` is one of the most basic types in Python. Just about anything is an `object`, so this doesn't actually tell you much. Since your function does `return self` it may be more useful to indicate explicitly that the return is of the same type as the object.
```suggestion
        MyLucas
```

Reply:
Noted.